### PR TITLE
Add @hemanth's AMA link

### PR DESCRIPTION
**Pull request by hemanth**
//cc @sindresorhus

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Derick Bailey](https://github.com/derickbailey/ama) - Developer and entrepreneur, [author](http://derickbailey.com/publications) and [screencaster](http://watchmecode.com).
 - [Andy Jiang](https://github.com/lambtron/ama/) - Customer Acquisition and Evangelist at [Segment](https://segment.com), maker of [things](http://blog.andyjiang.com).
 - [Benny Thomas](https://github.com/virtueme/ama/) - Full stack developer caring about children reading.
+- [Hemanth.HM](https://github.com/hemanth/ama/) - [h3manth](http://h3manth.com) A computer polyglot CLI, web and unix philosophy ❤️'r.
```
Comment (sindresorhus) [inline readme.md]: Needs a `.` after polyglot and the website mention should be more natural part of the text.

**Comment by sindresorhus** [comment]
No need to mention me on my own repos btw.

**Comment by hemanth** [comment]
Done. :+1:

**Comment by arthurvr** [comment]
> and the website mention should be more natural part of the text.

@hemanth :arrow_up: A username followed by a period isn't really a natural text.

**Comment by hemanth** [comment]
@arthurvr Hmm, yeah was thinking that it was link, should have been more like I blog at [h3manth.com](http://h3manth.com) anyway chucked that link altogether :P

File: readme.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Derick Bailey](https://github.com/derickbailey/ama) - Developer and entrepreneur, [author](http://derickbailey.com/publications) and [screencaster](http://watchmecode.com).
 - [Andy Jiang](https://github.com/lambtron/ama/) - Customer Acquisition and Evangelist at [Segment](https://segment.com), maker of [things](http://blog.andyjiang.com).
 - [Benny Thomas](https://github.com/virtueme/ama/) - Full stack developer caring about children reading.
-
```
Comment (arthurvr) [inline readme.md:45]: Don't remove this line :) Fixed it while merging.

Comment (hemanth) [inline readme.md:45]: :+1:

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Comment by hemanth** [comment]
I have asked few questions to @passy and indeed the epic question in your AMA ;) :+1:

**Comment by stoeffel** [comment]
And let me know, when you get an awesome answer (for awesome-ama-answers).

**Pull request closed without merging**